NEWS.md

```diff
@@ -1,5 +1,7 @@
 # r2dii.analysis (development version)
 
+* `target_market_share()` now fails if input data has inconsistent values in the `loan_size_*_currency` (#279)
+
```

Comment:
The error was already implemented, right? If all we did is to add tests, then there is no user-facing change to report in NEWS.